### DISCUSSION TITLE
Subscriptions with PubSub class instance

### DISCUSSION THREAD

**Discussion by vicktor-kiprotich** [enisdenjo/graphql-ws#279] [Q&A] [answered]
With graphql-ws would it be necessary to have PubSub implementation  from libraries such as one with redis and amqp or RabbitMQ?, when in production stage.

**Comment by enisdenjo** [comment, marked as answer]
It is completely up to you and how you handle subscriptions.

I personally don't use any of them, I rely on Postgres built-in notification system with [PostGraphile](https://www.graphile.org/postgraphile/) on a vanilla HTTP NodeJS server powered by `graphql-ws`.
